### ISSUE TITLE
Python 3 Support

### ISSUE THREAD

**Issue by alep** [anossov/openrtb#12]
Hello, I was wondering if you are planning any python3 support? or if you know if there are any issues (aside from syntax) which would be hard to change?

**Comment by anossov** [comment]
I'll see what I can do. Shouldn't be hard.

**Comment by alep** [comment]
Thanks, I run `2to3` and it seems to work, only one test failing:

```python
class TestFields(unittest.TestCase):
    ...
    def test_bad_utf8(self):
        self.assertEqual(openrtb.base.String('x\xff'), 'x')
```

which I guess it's because `2to3` changes unicode to str so this the following function (result of running `2to3` doesn't work correctly:

```python
def String(value):
    if isinstance(value, str):
        return value
    if isinstance(value, str):
        return value.decode('utf-8', errors='ignore')

    return str(value)
```

I was wondering if there are any issues with encoding and the RTB protocol which python3 might brake.

Cheers!

**Comment by NorthIsUp** [comment]
the "hardest" part will be dealing with the ObjectMeta class. So `six` will probably have to be added as a dependency.

**Comment by anossov** [comment]
Resolved by merging #17
